flytekit/clis/sdk_in_container/init.py

```diff
@@ -0,0 +1,60 @@
+import click
+from cookiecutter.main import cookiecutter
+from cookiecutter.prompt import read_user_variable
+
+
+@click.group("init")
+def init():
+    """
+    Create flyte-ready projects.
+    """
+    # TODO: Add `interactive` boolean flag to allow for programmatic access to this subcommand.
+    pass
+
+
+@click.command("example")
+def generate_simple_example():
```

Comment:
Can we not create dynamic
Commands that automatically create different examples

Reply:
Can you say more? Maybe give an example.